packages/plugin-loyalties-api/src/models/definitions/common.ts

```
@@ -61,6 +61,7 @@ export interface IScoreParams {
   toDate: String;
   page?: number;
   perPage?: number;
+  campaignId?: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)